docs/js/main.js

```diff
@@ -54,15 +62,28 @@ modalCloseButton.addEventListener("click", () => {
 });
 
 // Functions
+function validateSearch(recipeInput) {
+    const acceptedLetter = /[a-zA-Z]/;
+    if (recipeInput.length === 0) {
+       searchMessages.innerText = "You didn't input anything!";
+    } else if (!recipeInput.match(acceptedLetter)) {
+       searchMessages.innerText = "Search for a food ingredient, no numbers of special characters needed!";
+   } else {
+    return recipeInput;
+    }
+};
+
 async function getData() {
     // create fetch url with user-entered search term
-    let recipeInput = document.getElementById("search-bar").value.trim();
+    let recipeInput = searchBarInput.value.trim();
     let url = `https://tasty.p.rapidapi.com/recipes/list?from=0&size=20&q=${recipeInput}`;
     // fetch recipes
     const res = await fetch(url, options);
     const data = await res.json();
     recipes = data.results;
+    return recipes;
```

Comment:
We don't need to return the recipes anymore because in the line above it sets the global variable recipes which we need for the createModal function, too.